contrib/containers/ci/Dockerfile

```diff
@@ -77,8 +77,7 @@ RUN apt-get update && apt-get install $APT_ARGS \
     wine-stable \
     wine32 \
     wine64 \
-    xorriso \
-    && rm -rf /var/lib/apt/lists/*
```

Comment:
why change this?

Reply:
Based on what I'm familiar with, running `rm -rf /var/lib/apt/lists/*` is done to reduce the size of the layer but Dash's Dockerfiles only runs it on the last invocation of `apt-get` instead of after _every_ invocation (see an invocation without the removal [here](https://github.com/dashpay/dash/blob/1d6990a/contrib/containers/ci/Dockerfile#L33)). 

Keeping that in mind, since this is no longer the last _manual_ invocation, the call is moved [here](https://github.com/dashpay/dash/blame/008461341fc7d2d54acd0aaf3578089539428f7a/contrib/containers/ci/Dockerfile#L110).

It seems like it would make more sense to add that line into every invocation but that's outside the scope of this PR.